bundle/manifests/rhoai-operator-webhook-service_v1_service.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
```

Comment:
There is a question about this file https://github.com/red-hat-data-services/rhods-operator/pull/361#issuecomment-2365226293

Reply:
@VaishnaviHire Can you please confirm that we should keep this file or not in PR ?